doc/api/errors.md

```diff
@@ -2980,6 +2980,14 @@ import 'package-name'; // supported
 
 `import` with URL schemes other than `file` and `data` is unsupported.
 
+<a id="ERR_UNSUPPORTED_RESOLVE_REQUEST"></a>
+
+### `ERR_UNSUPPORTED_RESOLVE_REQUEST`
+
+Calling `import.meta.resolve` with a bare specifier outside of a `file:` module
+is unsupported. Calling `import.meta.resolve` with a relative URL from a `data:`
+module is unsupported.
```

Comment:
```suggestion
is unsupported. Calling `import.meta.resolve` with a relative URL or bare specifier
from a `data:` module is unsupported.
```

Right?

Reply:
It's already covered by the previous sentence: Calling `import.meta.resolve` with a bare specifier outside of a `file:` module is unsupported.

Reply:
I find this paragraph very confusing. Maybe it would be clearer if we made it affirmative instead?

> - From modules loaded from `file:` URLs, `import.meta.resolve` can resolve absolute URLs, relative URLs, and bare specifiers.
> - From modules loaded from `https:` URLs, `import.meta.resolve` can resolve absolute URLs and relative URLs.
> - From modules loaded from `data:` URLs, `import.meta.resolve` can resolve only absolute URLs.

Or whatever the correct support matrix is.